frontend/src/components/common/Toasts.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
```

Comment:
you're making reusable components you are a DEMON 👏

Reply:
### thanks, never expect less from me